src/index.html

```diff
@@ -15,13 +15,391 @@
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap"
+    />
     <link
       rel="stylesheet"
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
```

Comment:
please pay attention to this rule and format your HTML code

![image](https://github.com/mate-academy/layout_catalog/assets/108538921/37bfc7ca-48ed-4423-93a2-075f2435eaf5)